infrastructure/lib/atat-web-api-stack.ts

```diff
@@ -34,125 +34,91 @@ export class AtatWebApiStack extends cdk.Stack {
       method: HttpMethod.GET,
       handlerPath: packageRoot() + "/api/portfolioDrafts/portfolio/getPortfolioStep.ts",
     });
 
-    const createFundingStep = new ApiDynamoDBFunction(this, "CreateFundingStep", {
-      resource: funding,
+    const createFundingStep = new ApiDynamoDBFunction(this, "CreateFundingStepFunction", {
       table: table,
       method: HttpMethod.POST,
       handlerPath: packageRoot() + "/api/portfolioDrafts/funding/createFundingStep.ts",
     });
 
-    const getFundingStep = new ApiDynamoDBFunction(this, "GetFundingStep", {
-      resource: funding,
+    const getFundingStep = new ApiDynamoDBFunction(this, "GetFundingStepFunction", {
       table: table,
       method: HttpMethod.GET,
       handlerPath: packageRoot() + "/api/portfolioDrafts/funding/getFundingStep.ts",
     });
 
+    // The API spec, which just so happens to be a valid CloudFormation snippet (with some actual CloudFormation
+    // in it) gets uploaded to S3
+    const apiAsset = new s3asset.Asset(this, "ApiSpecAsset", {
+      path: "./atat_provisioning_wizard_api.yaml",
+    });
+
+    // And now we include that snippet as an actual part of the template using the AWS::Include Transform. This
+    // is where the two previous pieces come together. Now, all the Fn::Sub and other functions will be resolved
+    // which means that we can reference the ARNs of the various functions.
+    const data = cdk.Fn.transform("AWS::Include", { Location: apiAsset.s3ObjectUrl });
+
+    // And with the data now loaded from the template, we can use ApiDefinition.fromInline to parse it as real
+    // OpenAPI spec (because it was!) and now we've got all our special AWS values and variables interpolated.
+    const restApi = new apigw.SpecRestApi(this, "AtatSpecTest", {
+      apiDefinition: apigw.ApiDefinition.fromInline(data),
+      endpointTypes: [apigw.EndpointType.REGIONAL],
+      parameters: {
+        endpointConfigurationTypes: apigw.EndpointType.REGIONAL,
+      },
+    });
     // TODO: getPortfolioDraft
     // TODO: getApplicationStep
     // TODO: createApplicationStep
     // TODO: submitPortfolioDraft
-    addTaskOrderRoutes(this, restApi);
+    addTaskOrderRoutes(this);
   }
 }
-
-function addTaskOrderRoutes(scope: cdk.Stack, restApi: apigw.RestApi) {
-  const taskOrderFiles = restApi.root.addResource("taskOrderFiles");
-  const taskOrderId = taskOrderFiles.addResource("{taskOrderId}");
+function addTaskOrderRoutes(scope: cdk.Stack) {
   const taskOrderManagement = new TaskOrderLifecycle(scope, "TaskOrders");
-  const createTaskOrderFile = new ApiS3Function(scope, "CreateTaskOrderFile", {
-    resource: taskOrderFiles,
+  const uploadTaskOrder = new ApiS3Function(scope, "UploadTaskOrderFunction", {
     bucket: taskOrderManagement.pendingBucket,
     method: HttpMethod.POST,
-    handlerPath: packageRoot() + "/api/taskOrderFiles/createTaskOrderFile.ts",
+    handlerPath: packageRoot() + "/api/taskOrderFiles/uploadTaskOrder.ts",
```

Comment:
Make sure these names are correct based on the current state of Jason's PR to rename them. I don't remember if that has been merged yet or not.

Reply:
These are the renamed files, don't worry.